packages/components/src/internal/renderers/FileColumnRenderer.tsx

```diff
@@ -49,14 +47,24 @@ const getFileDisplayValue = (rawDisplayValue: string): FileProp => {
 
 export const getAttachmentCardProp = (
     data?: any,
-    colRangeUri?: string,
+    isFileLink?: boolean,
     onRemove?: (attachment: IAttachment) => void
 ): AttachmentCardProps => {
     if (!data) return null;
 
-    const url = data.get('url');
-    const value = data.get('value');
-    const { filename, fileUnavailable } = getFileDisplayValue(data.get('displayValue'));
+    let url, value, display;
+    if (Iterable.isIterable(data)) {
+        url = data.get('url');
+        value = data.get('value');
+        display = getFileDisplayValue(data.get('displayValue'));
+    } else {
+        url = caseInsensitive(data, 'url');
+        value = caseInsensitive(data, 'value');
+        display = getFileDisplayValue(caseInsensitive(data, 'displayValue'));
+    }
```

Comment:
do we have control over the usages of this in a way that we could force the type of the data prop to be `Record<string, any>` and then make the callers be responsible for doing `data.toJS()` or something like that?

Reply:
I think if we do do that, we'd want to do it for all of our column/detail renderers.